deps/hiredis/ssl.c

```diff
@@ -312,6 +312,7 @@ static int redisSSLConnect(redisContext *c, SSL *ssl) {
     }
 
     hi_free(rssl);
+    SSL_free(ssl);
```

Comment:
Is this a local change or are we pulling a new hiredis here?
If we do, it's probably best as a separate PR that will be merged with a merge commit to avoid issues with subtree down the road.

Reply:
it copied in the hiredis fix.